resources/io/jenkins/infra/docker/pod-template.yml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: "v1"
+kind: "Pod"
+metadata:
+  labels:
+    jenkins: "agent"
+  annotations:
+    container.apparmor.security.beta.kubernetes.io/builder: unconfined
+    container.seccomp.security.alpha.kubernetes.io/builder: unconfined
+spec:
+  tolerations:
```

Comment:
This block is specific to the kubernetes cluster `publick8s` and IMHO it should be here. On release.ci we have a generic jnlp pod that include this section and then every Jenkinsfile inherit that specific job like [here](https://github.com/jenkins-infra/release/blob/master/Jenkinsfile.d/components/remoting#L14)

Reply:
I mean the tolerations block

Reply:
Good point ! It looks like a PR is incoming for https://github.com/jenkins-infra/charts/blob/master/config/default/jenkins-infra.yaml#L85 :)
I'm removing the tolerations block from the shared library at the same time. Good point.

Reply:
Removed the `tolerationsù` block